config/boards/mba8mpxl-ras314.wip

```diff
@@ -0,0 +1,9 @@
+# MBa8MPxL-RAS314 with TQMa8MPQL
+BOARD_NAME="TQ8MP-RAS314"
```

Comment:
This should be `MBa8MP-RAS314`, no?

Reply:
You are [right](https://www.dict.cc/?s=right). But we have decided to use a shorter name.
The ASCII Art for boardname looks nice if it fits in one line.
![grafik](https://github.com/armbian/build/assets/73469302/3ac84dd6-8bb3-480e-ac49-071aa1670c63)